hooks/useContentRoute.ts

```diff
@@ -0,0 +1,35 @@
+import { usePathname } from 'next/navigation'
+import { chapters } from 'content'
+
+const getPathData = (pathName: string) => {
+  const pathArray = pathName.split('/').filter((p) => p)
+  const isLessonRoute = pathArray.length === 4
+
+  const lessonId = isLessonRoute ? pathArray.pop() : undefined
+  const lang = pathArray.shift()
+
+  const chapterId = pathArray.pop()
```

Comment:
In the case of `/en/chapters` wouldn't `chapterId` be set to `chapters` using this approach?